pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1420,11 +1435,15 @@ protected void handleAck(CommandAck ack) {
         final long consumerId = ack.getConsumerId();
 
         if (consumerFuture != null && consumerFuture.isDone() && !consumerFuture.isCompletedExceptionally()) {
+            Consumer consumer = consumerFuture.getNow(null);
```

Comment:
Since we have the `consumer` variable here, we can also change the next line to avoid doing `getNow()` again.

Reply:
Done